sdk/storage/azure-storage-blob/azure/storage/blob/_models.py

```diff
@@ -1302,3 +1304,52 @@ def __init__(self, error=None, is_fatal=False, description=None, position=None):
         self.is_fatal = is_fatal
         self.description = description
         self.position = position
+
+class BlobAudience(object):
+    """Audiences available for Blobs.
+
+    :keyword str value: The Azure Active Directory audience to use when forming authorization scopes.
+        For the Language service, this value corresponds to a URL that identifies the Azure cloud
+        where the resource is located. For more information please take a look at:
+        https://learn.microsoft.com/en-us/azure/storage/blobs/authorize-access-azure-active-directory
+        Please use one of the static constant members over creating a custom value unless you have
+        a specific scenario for doing so.
+    """
+
+    value: Optional[str] = None
+
+    def __init__(self, value: Optional[str]) -> None:
+        self.value = value
+
+    @staticmethod
```

Comment:
Use `@classmethod` instead. First param will be `cls` and you call that to construct the object.